### PR TITLE
Table unit tests

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Implemented Unit Tests for the tables view components.

Also, changed to useIntl() from passing as property.

### CODE REVIEW

File: webapp/src/components/table/tableHeaderMenu.test.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+jest.mock('../../mutator', () => ({
+    changeViewSortOptions: jest.fn(),
+    insertPropertyTemplate: jest.fn(),
+    changeViewVisibleProperties: jest.fn(),
+    duplicatePropertyTemplate: jest.fn(),
+    deleteProperty: jest.fn(),
+}))
+
+// jest.mock('mutator', () => {
```
Comment (harshilsharma63) [inline webapp/src/components/table/tableHeaderMenu.test.tsx:32]: Can we remove the comments?

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
@sbishel the build seems to be failing which also fails the table test.

**Review by hahmadia — APPROVED** [review]
Awesome! This sets a nice foundation for test. Thanks :)

**Pull request merged**